pkgs/build-support/trivial-builders/default.nix

```diff
@@ -363,16 +359,61 @@ rec {
     let
       runtimeShell' = runtimeShell;
     in
-    { name
-    , text
-    , runtimeShell ? runtimeShell'
-    , runtimeInputs ? [ ]
-    , runtimeEnv ? null
-    , meta ? { }
-    , checkPhase ? null
-    , excludeShellChecks ? [ ]
-    , bashOptions ? [ "errexit" "nounset" "pipefail" ]
-    , ...
+    {
+      /* The name of the script to write.
+
+         Type: String
+       */
```

Comment:
Unfortunately the doc comments here aren't rendered into the manual. Can you follow the style from the very new https://github.com/NixOS/nixpkgs/pull/277534 instead?

Reply:
Oh they aren't? But we could wire them up, no?

Reply:
Yeah that could be done with `nixdoc`, just like with the `lib` docs.